configure

```diff
@@ -179,55 +179,28 @@ do
     esac
 done
 
-if [ ${enable_boot} = 'yes' ]; then
-    schedules="${schedules} boot"
-else
-    schedules_not="${schedules_not} boot"
-fi
+check_sched_enabled()
+{
+    if eval [ "\"\${enable_${1}}\"" = \'yes\' ]; then
+        schedules="${schedules} ${2}"
+    else
+        schedules_not="${schedules_not} ${2}"
+    fi
+}
 
```

Comment:
thank you for this, you can also add yourself to the CHANGELOG. ("configure & Makefile cleanup ...")
I will publish a new release on 15/8